cfr.yaml

```diff
@@ -104,3 +104,29 @@
     id: openURL
     patterns:
       - "*://*/*"
+- id: INFOBAR_DEFAULT_AND_PIN_87
+  groups:
+    - cfr
+  content:
+    category: cfrFeatures
+    bucket_id: INFOBAR_DEFAULT_AND_PIN_87
+    text:
+      string_id: default-browser-notification-message
+    type: global
+    buttons:
+      - label:
+          string_id: default-browser-notification-button
+        action:
+          type: PIN_AND_DEFAULT
+        primary: true
+        accessKey: P
+  trigger:
+    id: defaultBrowserCheck
+  template: infobar
+  frequency:
+    lifetime: 100
+    custom:
+      - period: 3024000000  # 86400000 * 35 (35 days to ms)
+        cap: 1
+  # Not default browser and user dismissed the default browser prompt
+  targeting: "!isDefaultBrowser && !'browser.shell.checkDefaultBrowser'|preferenceValue && isMajorUpgrade != true"
```

Comment:
nit: let's do `!isMajorUpgrade`

Reply:
Doing explicit `!= true` to also cover the scenario of the attribute not being defined.